oat-sa/extension-tao-testqti
# Fix html entities name when insert on test

**Related to**

https://oat-sa.atlassian.net/browse/UDI-40

https://oat-sa.atlassian.net/browse/BOSA-37

**Description**

When you try to add some item, that have a name without English characters like `Torbørg`, to a test, the display name is not properly rendered

**How to test**

- Go to test and create one
- Insert some item which contains HTML entities in it name
- After insert, see if the name is correctly displayed
